dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -405,6 +410,46 @@ def build_docs(
     sys.exit(process.returncode)
 
 
+@main.command(name="publish-docs")
+@click.option("-d", "--disable-checks", help="Disables extra checks.", is_flag=True)
+@click.option("-s", "--override-versioned", help="Overrides versioned directories.", is_flag=True)
+@click.option(
+    "-s", "--airflow-site-directory", help="Local directory path of cloned airflow-site repo.", required=True
+)
+@click.option(
```

Comment:
I think this can be moved to "common_options" - same option is now used in `build_docs` so this can be made DRY